storage/grpc_client.go

```diff
@@ -1724,31 +1724,31 @@ func (w *gRPCWriter) uploadBuffer(recvd int, start int64, doneReading bool) (*st
 		}
 
 		// Done sending data (remainingDataFitsInSingleReq should == true if we
```

Comment:
Maybe clarify that we are done sending data in the provided buffer, but there are two separate cases depending on whether there is more data to send in the object overall. It's a little confusing as currently written.

Reply:
Modified the wording, does it make sense now?